lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -33,7 +33,7 @@
 
 // for specificity reasons this should be called using a custom class on the <body> tag
 // e.g. `<body className="nx-body nx-body--my-app">`
-@mixin font-size($custom-font-size) {
+@mixin nx-font-size($custom-font-size) {
```

Comment:
This is a breaking change.  I don't think we want to go to 2.0.0 over this.  Especially considering that less than half of our mixin names begin with `nx-`.

Reply:
> This is a breaking change. I don't think we want to go to 2.0.0 over this. Especially considering that less than half of our mixin names begin with `nx-`.

Right, reverted here: https://github.com/sonatype/sonatype-react-shared-components/pull/92/commits/f250f06da45d57fa575fddb0be981a3ea3eb2221